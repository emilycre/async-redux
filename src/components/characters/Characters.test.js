import React from 'react';
import { shallow } from 'enzyme';
import Characters from './Characters';

describe('Charactersss component', () => {
  it('Renders characters', () => {
    const wrapper = shallow(<Characters characters={[{ id: '0001', name: 'Appa', img: '/img' }]} />);
    expect(wrapper).toMatchSnapshot();
  });
});
