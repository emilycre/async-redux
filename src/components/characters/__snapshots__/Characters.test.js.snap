// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Charactersss component Renders characters 1`] = `
<ul>
  <li
    key="0001"
  >
    <Character
      character={
        Object {
          "id": "0001",
          "img": "/img",
          "name": "Appa",
        }
      }
    />
  </li>
</ul>
`;
